test/fixtures/invalid.js

```diff
@@ -75,6 +75,12 @@ var APP;
 		// eslint-disable-next-line no-new-wrappers
 		bar = new String( 'events' );
 
+		// eslint-disable-next-line max-len
+		bar = 'This is a long string that is indeed so long that it breaches the line length rules and thus would trigger a warning were it not for the over-ride.';
+
+		// eslint-disable-next-line max-len
+		// This is a ludicrously long comment that is so long that one can't help but wonder as to whether the writer is a fool or merely testing other developers.
```

Comment:
I suspect one or both of these cases aren't working properly. If they were working (as in, they fail without the inline comment), then the build should've been failing given this branch doesn't current add the expected warnings to `invalid-results.json`.

For the string, it not failing without the comment makes sense, given the `ignoreStrings: true` rule. However, I'm unsure as to why the long comment is also passing.